call out cpu and ram are only factors in cluster efficiency (#1070)
kubecost/docs: using-kubecost/navigating-the-kubecost-ui/efficiency-dashboard.md

![Efficiency Dashboard](/images/efficiency-dashboard-1.png)

> The Kubernetes Efficiency Dashboard  introduced in Kubecost 2.3 is designed to give Kubecost users a powerful tool for identifying the wasted spend coming from the clusters, nodes, and workloads in your Kubernetes environment.

Reviewer: ```suggestion
The Kubernetes Efficiency Dashboard introduced in Kubecost 2.3 is designed to give Kubecost users a powerful tool for identifying the wasted spend coming from the clusters, nodes, and workloads in their Kubernetes environment.
```

> These Efficiency Reports provides you with a single-pane of glass to understand your waste across all of your Kubernetes clusters. Particularly valuable is its ability to help  you understand and see the cost of idle resources on a per-cluster basis in a multi-cluster federation.

Reviewer: ```suggestion
These Efficiency Reports provide users with a single pane of glass to understand waste across all of their Kubernetes clusters. Particularly valuable is its ability to help  users understand and see the cost of idle resources on a per-cluster basis in a multi-cluster federation.
```

For example, in a pod with a single container which requests 2 Gi of memory but only uses 1 Gi, there is 1 Gi of workload idle which is assigned a cost. **This is a new definition of ‘idle’ for Kubecost.**
- _**Infra Idle**_ - Infra Idle (Infrastructure Idle) is defined as the difference between the cost of allocated resources and the cost of the hardware on which they run. Allocation is defined as the max of usage and request.
For example, a Kubernetes node which has 32 Gi of usable memory runs two pods each with one container. The first pod requests 2 Gi of memory but only uses 1 Gi. The second pod specifies no requests but uses 3 Gi. There is 27 Gi of Infra Idle which is assigned a cost. This is the same definition of ‘idle’ as used in the Kubecost Allocations page (see more details [here](/using-kubecost/navigating-the-kubecost-ui/cost-allocation/efficiency-idle.md)).
- _**Total Idle**_ - The sum of Infra Idle and Workload Idle.

> - _**Cluster Efficiency**_ - The ratio of the cost of used resources to the total cost of all resources. (cost of resources used) / (total cost of resources). This is the efficiency metric presented in the upper right-hand corner of the Overview page. **Note:** Today, 'Cluster Efficiency' only considers CPU and RAM in its calculations. We are working to integrate GPU and Storage metrics into the efficiency calculation in a future release.

Reviewer: ```suggestion
- _**Cluster Efficiency**_ - The ratio of the cost of used resources to the total cost of all resources. (cost of resources used) / (total cost of resources). This is the efficiency metric presented in the upper right-hand corner of the Overview page. **Note:** Today, 'Cluster Efficiency' only considers CPU and RAM in its calculations. GPU and Storage metrics will be factored into the efficiency calculation in a future release.
```
